regolith/helpers/l_projectahelper.py

```diff
@@ -22,29 +22,36 @@
 TARGET_COLL = "projecta"
 HELPER_TARGET = "l_projecta"
 
+ACTIVE_STATI = ["proposed", "started"]
+
 
 def subparser(subpi):
     subpi.add_argument("--all", action="store_true",
-                       help="Lists all projecta that have not ended"
+                       help="Lists all projecta in general"
+                       )
+    subpi.add_argument("-c", "--current", action="store_true",
+                       help="Lists all active projecta")
+    subpi.add_argument("-v", "--verbose", action="store_true",
+                       help='increase verbosity of output'
                        )
-    subpi.add_argument("-v", "--verbose", action="store_true", help='increase verbosity of output')
     subpi.add_argument("-l", "--lead",
                        help="Filter milestones for this project lead"
                        )
     subpi.add_argument("-p", "--person",
                        help="Filter milestones for this person whether lead or not"
                        )
     subpi.add_argument("-e", "--ended", action="store_true",
-                       help="Lists projects that have ended. Use the -d and -r flags to specify"
-                            "from one date and how many days"
+                       help="Lists projecta that have ended. Use the -d and -r flags to specify"
+                            " up to what date and how many days around that"
                        )
     subpi.add_argument("-d", "--date",
                        help="projecta with end_date within RANGE before this date will be listed."
-                            "Default is today"
+                            " Default is today"
+                            " Be wary some projecta do not have a specified end date"
```

Comment:
what is a user supposed to do with this information?  Give the users useful information that helps guide their actions.  I would delete this, but perhaps have a conversation about what your intent was here first?"

Reply:
The criteria for it to show up in -e is that it has to 1. have some sort of finished/submitted/cancelled string typed into 'status' and 2. the date has to fall in the range specified.

So since some projecta don't have the end_date entry, they won't come up in the search.

Reply:
almost., it should be status = finished only.

If status = finished and there is no end-date, then print a warning.

Reply:
in the warning, ask people to add an end date and remind people to use ``f_prum`` which automatically sets status to finished and adds an end date.

Reply:
what do you think about prums with status = cancelled?

Reply:
I'll set it up to print out an error_projecta list when you run -e. thought I feel like not many people would be using -e in general, so should this be something brought up to have people fix?

while getting the code to work the first time around, I did end up with a list of improperly formatted prums, so I already know who should fix status and end_date at least.

Reply:
cancelled means they weren't finished but are not active any more, but there may be info in there that we don't want to lose per se, so it should disappear off all lists basically, unless we want to recover them later which we could do using the kv filter.

Yes, you are right on the error code, but we will see the error list if it is run at group meeting....so people will know to fix it.  Over time this error rate will go down as people start using f_prum, which didn't exist before.

Reply:
ok, so current functionality won't show cancelled or anything else under finished prum searches